rest/bulk_api.go

```diff
@@ -295,8 +285,11 @@ func (h *handler) handleDumpChannel() error {
 // 	 ]
 // }
 func (h *handler) handleBulkGet() error {
-	includeRevs := h.getBoolQuery("revs")
 	includeAttachments := h.getBoolQuery("attachments")
+	revsLimit := 0
+	if h.getBoolQuery("revs") {
+		revsLimit = int(h.getIntQuery("revs_limit", math.MaxInt32))
```

Comment:
The type conversion is a bit hard to follow here - passing a MaxInt32 to a method expecting uint64, then casting a returned uint64 back to int.  I think it should all build ok (even for a 32-bit build), but I'm wondering whether revsLimit could just be managed as an unsigned int, since negative values aren't valid anyway (I don't think?).